src/registries/menus/topbar_menu_registry.ts

```diff
@@ -103,11 +104,13 @@ topbarMenuRegistry
     name: ACTIONS.REMOVE_ROWS_NAME,
     sequence: 80,
     action: ACTIONS.REMOVE_ROWS_ACTION,
+    isVisible: (env: SpreadsheetChildEnv) => CAN_REMOVE_COLUMNS_ROWS("ROW", env),
```

Comment:
the helper should be at the same place as the other menu item helpers (`lenu_item_action.ts`), and be imported/used like the others  `ACTION.CAN_REMOVE_COLUMNS_ROW`